src/main/java/com/training/web/pages/flipkart/ProductPage.java

```diff
@@ -1,53 +1,46 @@
 package com.training.web.pages.flipkart;
 
+import com.codeborne.selenide.Condition;
 import com.training.basepages.FlipkartBasePage;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
 
 public class ProductPage extends FlipkartBasePage {
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
-    private By brandName = By.className("G6XhRU");
-    private By productName = By.className("B_NuCI");
-    private By size = By.id("swatch-0-size");
-    private By cart = By.xpath("//button[text()='ADD TO CART']");
-    private By price = By.xpath("//div[contains(@class,'_30jeq3')]");
-    private By sizeList = By.xpath("//a[contains(@class,'_1fGeJ5')]");
-
-    public ProductPage(WebDriver driver) {
-        super(driver);
-    }
+    private String brandName = "//*[@class='G6XhRU']";
+    private String productName = "//*[@class='B_NuCI']";
+    private String size = "//*[@id='swatch-0-size']";
+    private String cart = "//button[text()='ADD TO CART']";
+    private String price = "//div[contains(@class,'_30jeq3')]";
+    private String sizeList = "//a[contains(@class,'_1fGeJ5')]";
 
     public ProductPage clickSize(String shoeSize) {
         String shoeSizeXpath = String.format("//a[text()=%1$s]", shoeSize);
```

Comment:
define this globally without format and then use it in line 22 with String.format